shared/NavLink.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+
+interface NavLinkProps {
+  href: string;
+  children: React.ReactNode;
+}
+
+const NavLink: React.FC<NavLinkProps> = ({ href, children }) => {
+  return (
+    <Link
+      href={href}
+      className="text-black hover:text-purple-600 hover:underline"
```

Comment:
- [ ] custom variant and color (https://nextui.org/docs/customization/custom-variants)